client/asset/dcr/dcr.go

```diff
@@ -505,44 +510,36 @@ func (dcr *ExchangeWallet) FundOrder(value uint64, nfo *dex.Asset) (asset.Coins,
 	return coins, err
 }
 
-// unspents fetches unspent outputs for the ExchangeWallet account.
+// unspents fetches unspent outputs for the ExchangeWallet account using rpc
+// RawRequest.
 func (dcr *ExchangeWallet) unspents() ([]walletjson.ListUnspentResult, error) {
-	walletUnspents, err := dcr.node.ListUnspentMin(0)
-	if err != nil {
-		return nil, fmt.Errorf("ListUnspentMin error: %w", err)
-	}
-
-	// Filter out the unspents for other accounts.
 	var unspents []walletjson.ListUnspentResult
-	for _, unspent := range walletUnspents {
-		if unspent.Account != dcr.acct {
-			continue
-		}
-		unspents = append(unspents, unspent)
-	}
-	return unspents, nil
+	// minconf, maxconf (rpcdefault=9999999), [address], account
+	params := anylist{0, 9999999, nil, dcr.acct}
+	err := dcr.nodeRawRequest(methodListUnspent, params, &unspents)
+	return unspents, err
 }
 
 // fund finds coins for the specified value. A function is provided that can
 // check whether adding the provided output would be enough to satisfy the
-// needed value.
-func (dcr *ExchangeWallet) fund(confs uint32,
-	enough func(sum uint64, size uint32, unspent *compositeUTXO) bool) (asset.Coins, error) {
+// needed value. Preference is given to selecting coins with 1 or more confs,
+// falling back to 0-conf coins where there are not enough 1+ confs coins.
+func (dcr *ExchangeWallet) fund(enough func(sum uint64, size uint32, unspent *compositeUTXO) bool) (asset.Coins, error) {
```

Comment:
Eliminating the confs arg is fine (we can bring it back easily if needed), but please call such changes out as well.

Reply:
Done.